component.json

```diff
@@ -12,6 +12,11 @@
                     "label": "WebHook URL",
                     "required": false
                 },
+                "hmac_key": {
+                    "viewClass": "OptionalRandomString",
```

Comment:
it is really very nice idea. however, I would love to see this feature working with minimal functionality. If we have some time left in the sprint we can do a 2nd iteration or plan it for later sprints.